keps/sig-api-machinery/4006-transition-spdy-to-websockets/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Transition from SPDY to Websockets
+kep-number: 4006
+authors:
+  - "@seans3"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-cli
+status: provisional
+# status other states: # |implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2023-05-15
+reviewers:
+  - "@aojea"
+  - "@ardaguclu"
+approvers:
+  - "@deads2k"
+  - "@jpbetz"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: Websockets
```

Comment:
we should have a more meaningful name, but I'm bad for names `KubectlWebsockets` or `ClientWebsockets` ??

Reply:
+1. If there is going to be a separate KEP for `kubectl port-forward` then maybe something like `ClientSubcommandWebsockets`?

Reply:
Added `ClientRemoteCommandWebsockets`.